x-pack/plugin/ml/src/internalClusterTest/java/org/elasticsearch/xpack/ml/integration/MlNodeShutdownIT.java

```diff
@@ -74,14 +74,16 @@ public void testJobsVacateShuttingDownNode() throws Exception {
         });
 
         // Call the shutdown API for the chosen node.
+        final SingleNodeShutdownMetadata.Type type = randomFrom(SingleNodeShutdownMetadata.Type.values());
+        final String targetNodeName = type == SingleNodeShutdownMetadata.Type.REPLACE ? randomAlphaOfLengthBetween(10, 20) : null;
         client().execute(
             PutShutdownNodeAction.INSTANCE,
             new PutShutdownNodeAction.Request(
                 nodeIdToShutdown.get(),
                 randomFrom(SingleNodeShutdownMetadata.Type.values()),
```

Comment:
```suggestion
                type,
```

Otherwise 7.x will suffer the problem of #77297

Reply:
Thanks and sorry for the noise! Tagging https://github.com/elastic/elasticsearch/pull/77304 so the PRs will be linked.